pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -89,3 +89,19 @@ def arange(start, stop, step):
         return torch.arange(start, stop, step, dtype=dtype)
 
     return arange
+
+
+@pytorch_funcify.register(Eye)
+def pytorch_funcify_eye(op, **kwargs):
+    dtype = getattr(torch, op.dtype)
+
+    def eye(N, M, k):
+        mjr, mnr = (M, N) if k > 0 else (N, M)
+        k_abs = abs(k)
+        zeros = torch.zeros(N, M, dtype=dtype)
+        if k_abs < mjr:
+            l_ones = min(mjr - k_abs, mnr)
```

Comment:
this could be `torch.min`

Reply:
So maybe better to use that? Same for abs